src/components/common/SearchBar/SearchBar.jsx

```diff
@@ -0,0 +1,53 @@
+import { Formik } from 'formik';
+import { Icon } from '../Icon';
+import {
+    SearchBarWrapper,
+    SearchButton,
+    SearchBarInput,
+} from './SearchBar.styled';
+
+
+const initialValues = {
+    search: '',
+};
+
+
+const SearchBar = () => {
+    const handleSubmit = ({search}, {resetForm}) => {
+        console.log(search);
+        resetForm();
```

Comment:
Are you sure you need to reset the form on submit?

Reply:
has removed